test/celeritas/ext/RootImporter.test.cc

```diff
@@ -399,3 +407,265 @@ TEST_F(RootImporterTest, em_params)
     EXPECT_VEC_EQ(expected_enum_string, enum_string);
     EXPECT_VEC_EQ(expected_value, value);
 }
+
+//---------------------------------------------------------------------------//
+TEST_F(RootImporterTest, sb_data)
```

Comment:
I think this test should move to the new (since you started this branch) `GeantImporter.test.cc` test. Then we don't have to have the data on the ROOT file.